### PR TITLE
Proofread NLP

### PR DESCRIPTION
**Pull request by goldmermaid**
In "approx-training.md", change from $P and N_1, \ldots, N_K$ to $P$ and $N_1, \ldots, N_K$ ;
In "sentiment-analysis-rnn.md", add "load_data_imdb" function;
In "word2vec-gluon.md", minor edits.

### CODE REVIEW

File: chapter_generative_adversarial_networks/gan.md [hunk 1]
```diff
@@ -174,3 +174,12 @@ train()
 ```
 
 ## Summary
+
+* Generative Adversarial Networks (GANs) composes of two deep networks, the generator and the discriminator.
```
Comment (astonzhang) [inline chapter_generative_adversarial_networks/gan.md]: No need to capitalize all the initials

File: chapter_generative_adversarial_networks/gan.md [hunk 2]
```diff
@@ -174,3 +174,12 @@ train()
...
+* The generator generates the image as much closer to the true image as possible to fool the discriminator, via maximizing the cross entropy loss, i.e. $ \max \log(D(\mathbf x'))$. 
+* The discriminator tries to distinguish the generated images from the true images, via minimizing the cross entropy loss, i.e. $ \min - y \log D(\mathbf x) - (1-y)\log(1-D(\mathbf x))$.
+
+
+## Reference
+
+[1] Ian Goodfellow, Jean Pouget-Abadie, Mehdi Mirza, Bing Xu, David Warde-Farley, Sherjil Ozair, Aaron Courville, and Yoshua Bengio. Generative adversarial nets. In Advances in neural information processing systems, 2672–2680. 2014.
```
Comment (astonzhang) [inline chapter_generative_adversarial_networks/gan.md]: Have we referred this paper in the main text (appending [1] at the end of a sentence)?

File: chapter_generative_adversarial_networks/gan.md [hunk 3]
```diff
@@ -174,3 +174,12 @@ train()
 ```
 
 ## Summary
+
+* Generative Adversarial Networks (GANs) composes of two deep networks, the generator and the discriminator.
+* The generator generates the image as much closer to the true image as possible to fool the discriminator, via maximizing the cross entropy loss, i.e. $ \max \log(D(\mathbf x'))$. 
+* The discriminator tries to distinguish the generated images from the true images, via minimizing the cross entropy loss, i.e. $ \min - y \log D(\mathbf x) - (1-y)\log(1-D(\mathbf x))$.
```
Comment (astonzhang) [inline chapter_generative_adversarial_networks/gan.md]: append a comma at the end of i.e.

File: chapter_generative_adversarial_networks/gan.md (hunk 2 above)
Comment (goldmermaid) [reply chapter_generative_adversarial_networks/gan.md]: We refer at "In 2014, a breakthrough paper introduced Generative adversarial networks (GANs) ...", while the number is [5] in the original version.

**Comment by astonzhang** [comment]
Could you resolve this conflicts? Thanks

**Comment by goldmermaid** [comment]
Resolved :)

**Comment by goldmermaid** [comment]
Conflicts resolved. Can we merge it now?

**Pull request merged**